osu.Game/Overlays/Profile/Sections/PaginatedProfileSubsection.cs

```diff
@@ -111,17 +109,20 @@ private void onUserChanged(ValueChangedEvent<APIUser> e)
 
         private void showMore()
         {
+            if (User.Value == null)
+                return;
+
             loadCancellation = new CancellationTokenSource();
 
             CurrentPage = CurrentPage?.TakeNext(ItemsPerPage) ?? new PaginationParameters(InitialItemsCount);
 
-            retrievalRequest = CreateRequest(CurrentPage.Value);
-            retrievalRequest.Success += UpdateItems;
+            retrievalRequest = CreateRequest(User.Value, CurrentPage.Value);
+            retrievalRequest.Success += items => UpdateItems(items, loadCancellation);
 
             api.Queue(retrievalRequest);
         }
 
-        protected virtual void UpdateItems(List<TModel> items) => Schedule(() =>
+        protected virtual void UpdateItems(List<TModel> items, CancellationTokenSource cancellationTokenSource) => Schedule(() =>
```

Comment:
The cancellation token source is now passed to this method as otherwise static analysis could not determine that the ambient `loadCancellation` field is not null.